build-tools/packages/build-tools/src/fluidBuild/tasks/taskFactory.ts

```diff
@@ -78,10 +78,16 @@ const executableToLeafTask: {
  * @param executable The command executable to find a matching task handler for.
  * @returns A `TaskHandler` for the task, if found. Otherwise `UnknownLeafTask` as the default handler.
  */
-function getTaskForExecutable(executable: string, context: BuildContext): TaskHandler {
+function getTaskForExecutable(
+	executable: string,
+	node: BuildPackage,
+	context: BuildContext,
+): TaskHandler {
 	const config = context.fluidBuildConfig;
 	const declarativeTasks = config?.declarativeTasks;
-	const taskMatch = declarativeTasks?.[executable];
+	const taskMatch =
+		node.pkg.packageJson.fluidBuild?.declarativeTasks?.[executable] ??
+		declarativeTasks?.[executable];
```

Comment:
What if a declarative task is defined in the root of the workspace? How does it interact with the ones declared in the config and those in the individual packages?

Reply:
For declarative task, it is a override.